specification/azsadmin/resource-manager/storage/Microsoft.Storage.Admin/preview/2016-05-01/storageaccounts.json

```diff
@@ -0,0 +1,441 @@
+                    }
+                }
+            },
+            "post": {
+                "x-ms-examples": {
+                    "Undelete a deleted storage account.": {
+                        "$ref": "./Examples/StorageAccounts/Undelete.json"
+                    }
+                },
+                "tags": [
+                    "StorageAccounts"
+                ],
+                "operationId": "StorageAccounts_Undelete",
+                "description": "Undelete a deleted storage account.",
+                "parameters": [
+                    {
+                        "$ref": "storage.json#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "storage.json#/parameters/ResourceGroupNameParameter"
+                    },
+                    {
+                        "$ref": "storage.json#/parameters/FarmIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/AccountIdParameter"
+                    },
+                    {
+                        "$ref": "storage.json#/parameters/ApiVersionParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK -- Undelete either has been performed or account was not deleted."
+                    }
+                }
+            }
+        }
+    },
+    "definitions": {
+        "StorageAccountState": {
+            "readOnly": true,
+            "description": "Storage account state.",
+            "type": "string",
+            "enum": [
+                "Creating",
+                "Succeeded",
+                "Suspended"
+            ],
+            "x-ms-enum": {
+                "name": "StorageAccountState",
+                "modelAsString": true
+            }
+        },
+        "StorageAccountType": {
+            "description": "Storage account type.",
+            "type": "string",
+            "enum": [
+                "Standard_LRS",
+                "Standard_GRS",
+                "Standard_ZRS",
+                "Standard_RAGRS",
+                "Premium_LRS"
+            ],
+            "x-ms-enum": {
+                "name": "StorageAccountType",
+                "modelAsString": true
+            }
+        },
+        "LocationStatus": {
+            "readOnly": true,
+            "type": "string",
+            "description": "Gets the status indicating whether the primary location of the storage account is available or unavailable.",
+            "enum": [
+                "Available",
+                "Unavailable"
+            ],
+            "x-ms-enum": {
+                "name": "LocationStatus",
+                "modelAsString": true
+            }
+        },
+        "StorageAccountOperation": {
+            "readOnly": true,
+            "description": "The permissions of storage account in WAC.",

```

Comment:
Double check the description `The permissions of storage account in WAC.` model seems representing various operations on storage account